README.md

```diff
@@ -178,6 +181,8 @@ Number of concurrent runs.
 
 ### `sourceMap`
 
+If you use custom `minify` function please read `minify` option section for handle source maps correctly.
```

Comment:
`If you use custom `minify` function please read `minify` option section for handle source maps correctly.` -> `If you use your own `minify` function please read the `minify` section for handling source maps correctly.`